src/include/duckdb/execution/operator/aggregate/physical_hash_aggregate.hpp

```diff
@@ -48,6 +48,9 @@ class PhysicalHashAggregate : public PhysicalSink {
 	//! Pointers to the aggregates
 	vector<BoundAggregateExpression *> bindings;
 
+    //! Map between payload index and input index for filters
+	unordered_map<Expression*,std::pair<bool,unordered_map<size_t,size_t>>> filter_map;
```

Comment:
This seems overly complicated; why not just add the filter one after the regular payload of an aggregate?